resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -0,0 +1,1877 @@
+        - name: antibody_catalog_number
+          label: Antibody catalog number
+          type: basic:string
+          required: false
+          hidden: "experiment.assay_type != 'chip-seq' && experiment.assay_type != 'chipmentation'"
+        - name: antibody_lot_number
+          label: Antibody lot number
+          type: basic:string
+          required: false
+          hidden: "experiment.assay_type != 'chip-seq' && experiment.assay_type != 'chipmentation'"
+    - name: sample_preparation_and_processing
+      label: Sample preparation and processing
+      group:
+        - name: sample_obtained_in_site
+          label: Sample obtained in site
+          type: basic:string
+          required: false
+        - name: sample_type
+          label: Sample type
+          type: basic:string
+          description: |
+            Source of biological material, e.g. fresh or FFPE.
+          required: false
+        - name: archival
+          label: Archival
+          type: basic:string
+          required: false
+          choices:
+            - label: Fresh
+              value: fresh
+            - label: Archival
+              value: archival
+        - name: sampling_time
+          label: Sampling time
+          type: basic:string
+          required: false
+          choices:
+            - label: Pretreatment
+              value: pretreatment
+            - label: Posttreatment
+              value: posttreatment
+        - name: tumor
+          label: Tumor
+          type: basic:string
+          description: |
+            Select what molecule your samples are from.
+          required: false
+          choices:
+            - label: RNA
+              value: rna
+            - label: DNA
+              value: dna
+        - name: normal
+          label: Normal
+          type: basic:string
+          description: |
+            Select what molecule your samples are from.
+          required: false
+          choices:
+            - label: RNA
+              value: rna
+            - label: DNA
+              value: dna
+        - name: date_of_preparation
+          label: Date of preparation
+          type: basic:date
+          required: false
+        - name: date_of_shipping
+          label: Date of shipping
+          type: basic:date
+          required: false
+        - name: date_received
+          label: Date received
+          type: basic:date
+          required: false
+        - name: platform
+          label: Platform
+          type: basic:string
+          description: |
+            Select by which platform the sample was prepared.
+          required: false
+          choices:
+            - label: RNA-seq
+              value: rna-seq
+            - label: DNA-seq
+              value: dna-seq
+            - label: Nanostring
+              value: nanostring
+            - label: Microarray
+              value: microarray
+            - label: Edge-seq
+              value: edge-seq
```

Comment:
What to do with this field? Field `platform` in `experiment` group above is not the same as this one.